rust/routee-compass/src/app/compass/config/frontier_model/road_class/road_class_service.rs

```diff
@@ -18,7 +24,7 @@ impl FrontierModelService for RoadClassFrontierService {
     ) -> Result<Arc<dyn FrontierModel>, FrontierModelError> {
         let service: Arc<RoadClassFrontierService> = Arc::new(self.clone());
         let road_classes = query
-            .get_config_serde_optional::<HashSet<String>>(&"road_classes", &"")
+            .get_config_serde_optional::<HashSet<u8>>(&"road_classes", &"")
```

Comment:
to implement the mapping action, when road_class_mapping is not None, do we replace this logic with the mapping logic, and then set "road classes" from the result of the mapping? seems like just a few more lines to add to the service here.